applications/metarhia.com/lib/files.js

```diff
@@ -0,0 +1,77 @@
+api.files = {};
+
+api.files.attachmentNameToFilePath = (
+  // transofrms attachmentName to file path in filesystem
+  attachmentName // name which can be got from uploadFile
+) => {
+  const prefix1 = attachmentName.slice(0, 2);
+  const prefix2 = attachmentName.slice(2, 4);
+  const fileName = attachmentName.slice(4);
+  const filePath = (
+    application.dir + '/files/' +
+    prefix1 + '/' + prefix2 + '/' + fileName
+  );
+  return filePath;
+};
+
+{
+  const impress = {};
+  impress.ALPHA_UPPER = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+  impress.ALPHA_LOWER = 'abcdefghijklmnopqrstuvwxyz';
+  impress.ALPHA = impress.ALPHA_UPPER + impress.ALPHA_LOWER;
+  impress.DIGIT = '0123456789';
+  impress.ALPHA_DIGIT = impress.ALPHA + impress.DIGIT;
+  
+  api.files.uploadFile = (
+    // Upload file to files folder.
+    options, // <Readable> | { inp: <Readable>, timeout: <string> }. 
+             // Readable is data of the file.
+             // timeout defines time after which remove file
+    callback // <Function> (err, downloadedCode).
+             // downloadedCoded represents file.
+            // Can be passed to downloadFile.
+  ) => {
+    let inp, timeout;
```

Comment:
input stream, inspired by [uploadFile](https://github.com/metarhia/Impress/blob/2bda37c4df9ecc1bca8433003817d92a7425cddb/lib/impress.files.js#L150) in Impress